hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsHttpOperation.java

```diff
@@ -339,9 +340,10 @@ public void processResponse(final byte[] buffer, final int offset, final int len
     if (this.requestId == null) {
       this.requestId = AbfsHttpConstants.EMPTY_STRING;
     }
+    responseHeaders = AbfsIoUtils.getResponseHeaders(connection);
     // dump the headers
     AbfsIoUtils.dumpHeadersToDebugLog("Response Headers",
-        connection.getHeaderFields());
+        responseHeaders);
```

Comment:
Why is this change required ?

Reply:
New method is introduced to print response headers which simply accepts the list of response headers